src/schema-converters/internalToExpanded.test.ts

```diff
@@ -336,12 +336,20 @@ describe('internalSchemaToExpanded', async function() {
           }
         ]
       };
-      const standard = await internalSchemaToExpanded(internal);
-      assert.deepStrictEqual(standard, {
+      const expanded = await convertInternalToExpanded(internal);
+      const expectedDefinitions: any = RELAXED_EJSON_DEFINITIONS;
```

Comment:
```suggestion
      const expectedDefinitions: any = { ...RELAXED_EJSON_DEFINITIONS };
```

Reply:
Also do we need this `any` setting? Maybe we can drop it or set it as Partial<typeof lRELAXED_EJSON_DEFINITIONS>